tests/loader_test_vectors.rs

```diff
@@ -61,20 +67,64 @@ fn concat<'a>(left: &[&'a str], right: &[&'a str]) -> Vec<&'a str> {
         .collect()
 }
 
+/// Determines if the given file name is in the paths list.  This deals with platform
+/// path separator differences from '/' separators in the path list.
+#[inline]
+fn contains_path(paths: &[&str], file_name: &str) -> bool {
+    paths
+        .into_iter()
+        // TODO construct the paths in a not so hacky way
+        .map(|p| p.replace("/", &PATH_SEPARATOR.to_string()))
+        .find(|p| p == file_name)
+        .is_some()
+}
+
 fn load_file<L: Loader>(loader: &L, file_name: &str) -> IonResult<Vec<OwnedElement>> {
     // TODO have a better API that doesn't require buffering into memory everything...
     let data = read(file_name)?;
-    loader.load_all(&data)
+    let result = loader.load_all(&data);
+
+    // do some simple single value loading tests
+    let single_result = loader.load_one(&data);
+    match &result {
```

Comment:
I think I could've written it with a match on a pair of `(&result, &single_result)` or some such, but I don't think that would've been clearer in my opinion.